jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -50,24 +51,46 @@ default String dump()
     void dump(Appendable out, String indent) throws IOException;
 
     /**
-     * Utility method to implement {@link #dump()} by calling {@link #dump(Appendable, String)}
+     * Utility method to call dump to a {@link String}
      *
      * @param dumpable The dumpable to dump
      * @return The dumped string
+     * @see #dump(Appendable, String)
      */
     static String dump(Dumpable dumpable)
     {
         StringBuilder b = new StringBuilder();
+        dump(dumpable, b);
+        return b.toString();
+    }
+
+    /**
+     * Utility method to dump to an {@link Appendable}
+     *
+     * @param dumpable The dumpable to dump
+     * @param out The destination of the dump
+     */
+    static void dump(Dumpable dumpable, Appendable out)
+    {
         try
         {
-            dumpable.dump(b, "");
+            dumpable.dump(out, "");
+
+            out.append(KEY);
+            Runtime runtime = Runtime.getRuntime();
+            out.append("JVM: %s %s; Jetty: %s; CPUs: %d; mem(free/total/max)B:%,d/%,d/%,d".formatted(
+                System.getProperty("java.runtime.name"),
+                System.getProperty("java.runtime.version"),
+                Jetty.VERSION,
+                runtime.availableProcessors(),
+                runtime.freeMemory(),
+                runtime.totalMemory(),
+                runtime.maxMemory()));
```

Comment:
I'd format in MiB (so divide by 1024*1024).

I'd also see what OS it is, so sysprop `os.[name|version|arch]`, and also a UTC timestamp.

Reply:
Now looks like:
```
key: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +@ map, +> undefined
JVM: OpenJDK Runtime Environment 22+36-2370; OS: Linux amd64 6.5.0-35-generic; Jetty: 12.0.10-SNAPSHOT; CPUs: 20; mem(free/total/max): 4,022/4,096/6,144 MB
UTC: 2024-05-27T13:17:06.456Z; Europe/Rome: 2024-05-27T15:17:06.456 +0200
```